src/top/utop.el

```diff
@@ -645,7 +648,10 @@ it is started."
        (setq utop-completion nil))
       ;; A new possible completion
       ("completion"
-       (push argument utop-completion))
+       (catch 'done
+	 (dolist (prefix utop-completion-prefixes)
```

Comment:
The indentation seems off. `(dolist` should be indented because it's inside `catch`

Reply:
Fixed.